packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -425,6 +425,29 @@ export function editPost(post: Post) {
         ],
     });
 }
+export function getPostsByIds(ids: string[]) {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        let posts;
+
+        try {
+            posts = await Client4.getPostsByIds(ids);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: PostTypes.GET_POSTS_FAILURE, error},
+                logError(error),
+            ]));
+            return {error};
+        }
+
+        dispatch({
+            type: PostTypes.RECEIVED_POSTS,
+            data: {posts},
+        });
+
+        return {data: {posts}};
+    };
+}
```

Comment:
Let me know if we're not planning to use this code and we can remove it and the server-side.

Reply:
You can remove that endpoint